tests/lang/core.pk

```diff
@@ -30,6 +30,9 @@ def fn(p1, p2, p3) end
 assert(fn.name == "fn")
 assert(fn.arity == 3)
 
+## String functions
+assert((str_sub('c programming', 3, 11)) == 'programming')
```

Comment:
Our substring range should match like in other languages. (like in the example below)
```
substring('foobarbaz', 2, 3) = 'oba'

f o o b a r 
0 1 2 3 4 5
    ^---^
```